presentation/src/main/java/com/ddd4/dropit/presentation/ui/add/AddSharedViewModel.kt

```diff
@@ -67,12 +59,38 @@ class AddSharedViewModel @ViewModelInject constructor(
 
     val nextClick = SingleLiveEvent<Void>()
 
+    val isPermissionState = SingleLiveEvent<Boolean>()
+
+    val captureClick = SingleLiveEvent<Void>()
+
     val addComplete = SingleLiveEvent<Long>()
 
+    val isLittleState = MutableLiveData<Boolean>()
+    val isDontState = MutableLiveData<Boolean>()
+    val dateLittleText = MutableLiveData<String>()
+    val dateDontText = MutableLiveData<String>()
+
     fun setProgressValue(value: Int) {
         _progressValue.value = value
     }
 
+    fun onNextClicked() {
+        nextClick.call()
+        _nextButtonState.value = false
+    }
+
+    fun onImageClicked() {
+        if (permissionHelper.isCapturePermissionState()) {
+            captureClick.call()
+        } else {
+            isPermissionState.value = false
+        }
+    }
+
+    fun startCapture() {
+        captureClick.call()
+    }
+
     fun getCategoryItems() {
         viewModelScope.launch {
             when (val result = getCategoryUseCase()) {
```

Comment:
하나의 함수에 유스케이스가 여러개 들어가면 when 을 여러번 써야겠네요
result를 분기할 때 getValue 방식을 추천드립니다.

```suggestion
            try{
            val result = getCategoryUseCase().getValue() 
            ... UseCase... etc.
            } cath(e: Exception) {
             Timber.e(e)
          }
```